src/components/Currency/index.js

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { useTranslation } from "react-i18next";
+
+const Currency = props => {
```

Comment:
Currency component should have a way of configuring default currency. If my application is dealing with USD all the time I don't want to write each time <Currency value={...} type="USD"/> I would prefer to configure USD as default and write <Currency value={...} /> everywhere.